From 2fd96e9168ab391bb1454358dd74492e778d8b43 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Thu, 18 May 2023 19:24:36 +0300
Subject: [PATCH 30/30] solidrun: switch to per-cpu timer as tick provider

This commit paraphrases TI change to EVM:
11d0860 "configs: am64x/am62x_evm_a53_defconfig: Switch to per-cpu timer as tick provider"

In actuality this fixes a problem where all system delays were off by
about a factor of 8 :(

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 configs/am64x_a53_solidrun_defconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configs/am64x_a53_solidrun_defconfig b/configs/am64x_a53_solidrun_defconfig
index 7bb0169ad3..9750868ab6 100644
--- a/configs/am64x_a53_solidrun_defconfig
+++ b/configs/am64x_a53_solidrun_defconfig
@@ -151,9 +151,6 @@ CONFIG_SYSRESET=y
 CONFIG_SPL_SYSRESET=y
 CONFIG_SYSRESET_TI_SCI=y
 CONFIG_DM_THERMAL=y
-CONFIG_TIMER=y
-CONFIG_SPL_TIMER=y
-CONFIG_OMAP_TIMER=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_HOST=y
-- 
2.35.3

